cmd/sync/sync.go

```diff
@@ -3,22 +3,142 @@ package sync
 
 import (
 	"context"
+	"io"
+	"os"
+
+	mutex "sync" // renamed as "sync" already in use
 
 	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/fs"
 	"github.com/rclone/rclone/fs/config/flags"
+	"github.com/rclone/rclone/fs/hash"
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/rclone/rclone/fs/sync"
 	"github.com/spf13/cobra"
 )
 
 var (
 	createEmptySrcDirs = false
+	combined           = ""
```

Comment:
I would just make a LoggerOpt here pass that directly to `AddLoggerFlags`

Reply:
Made this change.